src/client-services/github.client-service.test.tsx

```diff
@@ -0,0 +1,69 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import {
+  axiosConfig,
+  getReposList,
+  getRepoInfo,
+} from '../client-services/github.client-service';
+
+jest.mock('axios');
+
+describe('Github client service test', () => {
+  const accessToken = 'testToken';
+  const mockResponse: AxiosResponse = {
+    data: [{ name: 'openbc-web' }],
+    headers: {
+      Authorization: `token ${accessToken}`,
+      'Content-Type': 'application/json',
+    },
+    config: {},
+    headers: {},
+    request: {},
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    process.env.GITHUB_API_KEY = 'testToken';
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+    delete process.env.GITHUB_API_KEY;
+  });
+
+  describe('axiosConfig', () => {
+    it('should return an Axios instance', async () => {
```

Comment:
I suggest changing it to the following

`  describe('axiosConfig', () => {
    it('should return an Axios instance', () => {
      axios.create = jest.fn().mockReturnValue(mockResponse);

      const axiosInstance = axiosConfig();

      expect(axiosInstance).toEqual(mockResponse);
    });
  });`
  
  The 2 issues resolved:
1) Error high light on mockReturnValue. It's because axios.create is not recognized as a jest.fn()
2) axios.create is not a async function. So, the jest mock should use mockReturnValue. That also means you should not use async or await.

Reply:
I didn't think about mocking axios.create as a jest.fn() first, thank you!

Reply:
I think keeping `AxiosInstance` type when initializing axiosInstance is ok though?